### PR TITLE
[profile] Build ContactScreen and add contact us button to ProfileScreen

### PR DESCRIPTION
**Pull request by EthanAuyeung**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
Implemented the designs and functionality for v2 Profile flow.

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy!"

## How to review
[//]: # "Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"
`npm start` and play around with the profile section of the app.

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # "Optional - copy links to any tutorials or documentation that was useful to you when working on this PR"

### Related PRs
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

[//]: # "This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation."
🥦 CC: @sauhardjain

### CODE REVIEW

File: src/screens/profile/ContactUsScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    fetchData();
+  }, [vendorUuid]);
+  return (
+    <SafeArea>
+      <StandardHeader>
+        <BackButton onPress={() => navigation.goBack()} />
+      </StandardHeader>
+      <MainProfileContainer>
+        <HeadingContainer>
+          <H2Heading>Contact Us</H2Heading>
```
Comment (allisonhongberkeley) [inline src/screens/profile/ContactUsScreen.tsx:35]: Contact Us header should be left-aligned (per v2 design).

File: src/screens/profile/ProfileScreen.tsx [hunk 2]
```diff
@@ -56,6 +66,29 @@ export default function ProfileScreen() {
...
+            <Icon2
+              name="phone"
+              size={25}
+              style={Styles.icon}
+              color={Colors.darkGray}
+            />
+          </LeftAlignContainer>
+          <LeftAlignContainer>
+            <ContactUsContainer>
+              <H4CardNavTab>Contact Us</H4CardNavTab>
```
Comment (allisonhongberkeley) [inline src/screens/profile/ProfileScreen.tsx]: Nit: add a space between the phone icon and "Contact us" text.

Comment (EthanAuyeung) [reply src/screens/profile/ProfileScreen.tsx]: Fixed

File: src/screens/profile/ContactUsScreen.tsx (hunk 1 above)
Comment (EthanAuyeung) [reply src/screens/profile/ContactUsScreen.tsx:35]: Sauhard said its okay to keep it center aligned but awaiting approval from danielle

File: src/navigation/stacks/ProfileStackNavigator.tsx [hunk 3]
```diff
@@ -1,7 +1,9 @@
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
 import React from 'react';
 import ProfileScreen from '../../screens/profile/ProfileScreen';
+import ContactUsScreen from '../../screens/profile/ContactUsScreen';
 import { ProfileStackParamList } from '../types';
+import { ContactUsContainer } from '../../screens/profile/styles';
```
Comment (sauhardjain) [inline src/navigation/stacks/ProfileStackNavigator.tsx]: unused import

File: src/screens/profile/ContactUsScreen.tsx
Comment (sauhardjain) [inline src/screens/profile/ContactUsScreen.tsx:1]: Multiple unused imports on this screen, let's get rid of these before merging.

File: src/screens/profile/ProfileScreen.tsx [hunk 4]
```diff
@@ -22,8 +26,14 @@ import {
 import StandardLogo from '../../components/common/StandardLogo';
 import StandardHeader from '../../components/common/StandardHeader';
 import { getVendor } from '../../database/queries';
+import Colors from '../../../assets/Colors';
+import { Styles } from '../../components/transactions/styles';
+import ContactUsScreen from './ContactUsScreen';
```
Comment (sauhardjain) [inline src/screens/profile/ProfileScreen.tsx]: unused import

File: src/screens/profile/ContactUsScreen.tsx [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+          <LeftAlignContainer>
+            <Body1Text>Location</Body1Text>
+            <H4CardNavTab>put address here lol</H4CardNavTab>
+          </LeftAlignContainer>
+        </Row>
+      </TouchableOpacity>
+      <MagentaButtonContainer>
+        <ButtonMagenta onPress={handleSignOut}>
+          <ButtonTextWhite>Email Us</ButtonTextWhite>
+        </ButtonMagenta>
```
Comment (sauhardjain) [inline src/screens/profile/ContactUsScreen.tsx]: Why does this button sign out the user? The expected behavior is to open up the email app and compose an email to "v4vdevelopment@gmail.com"

File: src/screens/profile/styles.ts [hunk 6]
```diff
@@ -23,3 +23,6 @@ export const LeftAlignContainer = styled.View`
   align-content: flex-start;
   padding-left: 29px;
 `;
+export const ContactUsContainer = styled.View`
+  margin-left: -156px;
+`;
```
Comment (sauhardjain) [inline src/screens/profile/styles.ts]: Don't use an absolute negative margin here — results in wrong styling on different screen sizes.

**Review by sauhardjain — CHANGES_REQUESTED** [review]
Good work but there's a few issues. Left some comments and also saw this styling error on my device — can you fix this?

IMG_3E7CFFC76D28-1

File: src/screens/profile/styles.ts (hunk 6 above)
Comment (EthanAuyeung) [reply src/screens/profile/styles.ts]: Fixed

File: src/screens/profile/ContactUsScreen.tsx (hunk 5 above)
Comment (EthanAuyeung) [reply src/screens/profile/ContactUsScreen.tsx]: Fixed

File: src/navigation/stacks/ProfileStackNavigator.tsx (hunk 3 above)
Comment (EthanAuyeung) [reply src/navigation/stacks/ProfileStackNavigator.tsx]: Fixed

File: src/screens/profile/ContactUsScreen.tsx
Comment (EthanAuyeung) [reply src/screens/profile/ContactUsScreen.tsx:1]: Fixed

File: src/screens/profile/ProfileScreen.tsx (hunk 4 above)
Comment (EthanAuyeung) [reply src/screens/profile/ProfileScreen.tsx]: Fixed

**Comment by EthanAuyeung** [comment]
Just pushed a new commit ready for re-review @sauhardjain

**Review by sauhardjain — APPROVED** [review]
lgtm!
